cli/lib/exec/open.js

```diff
@@ -15,6 +15,12 @@ const { exitWithError } = require('../errors')
  * @returns {string[]} list of CLI arguments
  */
 const processOpenOptions = (options = {}) => {
+  // In addition to setting the project directory, setting the project option
+  // here ultimately decides whether cypress si run in global mode or not.
```

Comment:
SUPER helpful comment, tiny typo.

```suggestion
  // here ultimately decides whether cypress is run in global mode or not.
```